funcs_scripts.php

```diff
@@ -263,7 +281,7 @@ function is_gha_repository()
  * Determine if the module being processed is "TOOLING"
  *
  * Example usage:
- * is_gha_repository()
+ * is_tooling()
```

Comment:
This and the same below are unrelated fixes to phpdocs. Looks like it was just a copy/paste mistake